### PR TITLE
Removed references to Super Awesome support.

### PR DESCRIPTION
**Pull request by rlyne**

### CODE REVIEW

**Comment by david-richer-adsk** [comment]
I am not aware of the process but I expect you should only modify the english version.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/0b1642d8ad5ee6c1d5f89cb6ad779acb213c6cdd/index.html)

**Comment by rlyne** [comment]
> I am not aware of the process but I expect you should only modify the english version.

We had issues with the Isolation translations so they're all being kept in sync with the English translation for now.

File: docs/ja/trusted-solutions/tier1/getting_started/getting_started.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 layout: default
 title: Getting Started
 pagename: tier1-getting_started
-lang: ja
+lang: en
```
Comment (david-richer-adsk) [inline docs/ja/trusted-solutions/tier1/getting_started/getting_started.md:5]: Make sense?

Comment (rlyne) [reply docs/ja/trusted-solutions/tier1/getting_started/getting_started.md]: Got it.

**Review by david-richer-adsk — APPROVED** [review]

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/05874dde712b4ab76388fef07561e07aa128aa46/index.html)

**Pull request merged**